bin/dfx-stock-deploy

```diff
@@ -71,13 +71,25 @@ dfx-token-deploy \
   --network "$DFX_NETWORK" \
   --yes
 
+: Set up ckRED canisters
+dfx-token-import --prefix ckred_ --commit "$DFX_IC_COMMIT"
+dfx-token-deploy \
+  --prefix ckred_ \
+  --token ckRED \
+  --decimals 6 \
```

Comment:
Do we want 6 decimals? I guess this is copied from ckUSDC.
Standard is 8. If we want non-standard because it gives better test coverage, should we pick something other than 6?

I don't have a strong opinion, just wondering if you thought about it at all.

Reply:
Yep, I thought about it, and I think anything with a non-default value will work. But I’m not against changes; let’s use 7.